frontend/src/component/common/DialogFormTemplate/DialogFormTemplate.styles.tsx

```diff
@@ -16,16 +15,19 @@ export const StyledFormSection = styled('div')(({ theme }) => ({
 
 export const TopGrid = styled(StyledFormSection)(({ theme }) => ({
     display: 'grid',
-    gridTemplateAreas:
-        '"icon header" "icon project-name" "icon project-description"',
+    gridTemplateAreas: `
+        "icon header"
+        ".    project-name"
```

Comment:
This looks weird. Is it out formatting?

Reply:
Oh, yeah, I formatted it like this to make it visually match what the layout is. We can put it on a single line if you don't like it, but I think this is a useful way to show it.